adaptive_navigation/example/pubspec.lock

```diff
@@ -0,0 +1,161 @@
+# Generated by pub
+# See https://dart.dev/tools/pub/glossary#lockfile
```

Comment:
make sure not to check in any pubspec.lock's (see https://dart.dev/guides/libraries/private-files)

Reply:
Done.